App force closes because of no initialisation of mFragmentSelectedMap.

App force closes on running.

``
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void 
        android.util.SparseIntArray.append(int, int)' on a null object reference
        at swati4star.createpdf.activity.MainActivity.setTitleMap(MainActivity.java:275)
        at swati4star.createpdf.activity.MainActivity.initializeValues(MainActivity.java:170)
        at swati4star.createpdf.activity.MainActivity.onCreate(MainActivity.java:70) 
``

Checked out previous commits and found commit 6f3837c8cc47826ebf78fc7af5a9fe1beb26cc88 has removed the initialisation of ``SparseIntArray mFragmentSelectedMap``. 

Is there anything that I'm missing?

### Steps to reproduce
Set up clean project, compile and run.

### Expected behaviour
App should run without force closing.

### Actual behaviour
App force closes.

### Would you like to work on the issue?
Yes. Simple addition of the initialisation would do the trick.

--- a/app/src/main/java/swati4star/createpdf/activity/MainActivity.java
+++ b/app/src/main/java/swati4star/createpdf/activity/MainActivity.java
@@ -271,6 +271,7 @@ public class MainActivity extends AppCompatActivity
 
     // Hashmap for setting the mFragmentSelectedMap.
     public void setTitleMap() {
+        mFragmentSelectedMap = new SparseIntArray();
         mFragmentSelectedMap.append(R.id.nav_home, R.string.app_name);
         mFragmentSelectedMap.append(R.id.nav_camera, R.string.images_to_pdf);
         mFragmentSelectedMap.append(R.id.nav_qrcode, R.string.qr_barcode_pdf);

